### DISCUSSION TITLE
Display sub folders in Asset tables

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9171] [Ideas] [answered]
> Created by: Mats Mikkel Rummelhoff (mail@mmikkel.no) on 2015/03/09 00:12:31 +0000
> Votes at time of UserVoice import: 6

-----------------------------------

It'd be nice if clickable sub folders were displayed above files in the Asset tables.

It'd be a boost to usability, particularly in cases where you have folders with no files, only sub folders.

See attached file for mockup.

**Comment by khalwat** [comment]
I *think* this issue is what I'm looking for. Basically, clients can never seem to find the sub-folders for Assets, that little `>` just doesn't seem to be enough of a visual cue.

Something like this by default, or as a setting would be nice:

https://github.com/nystudio107/disclosedassets

**Comment by rungta** [comment]
Clients struggle with discovering, accessing and creating asset subfolders on most sites we deploy. And this is even after making them aware of the `>` icon and the right-click for "New subfolder" action.

**Comment by benface** [comment]
I was going to create this exact issue, when I saw this one from ~2017~ 2015. This would be awesome:

![folders](https://user-images.githubusercontent.com/1059139/53703024-19b5fc80-3ddb-11e9-8c0b-f3f3cf85642b.jpg)

**Comment by brandonkelly** [comment, marked as answer]
This is coming in Craft 3.8 and 4.4 🎉 (See the [Rethinking Volume Subfolders](https://craftcms.com/blog/rethinking-volume-subfolders) blog post and PR #12558 for more info.)

**Reply by alexanderloewe** [reply]
I would find it good to keep the current version with the navigation in the side column as well and merge both variants in one version.

This is especially useful for the behavior of large amounts of files

**Reply by brandonkelly** [reply]
3.8 and 4.4 are out with those changes 🎉 

@alexanderloewe See that [Rethinking Volume Subfolders](https://craftcms.com/blog/rethinking-volume-subfolders) blog post for an explanation of why we are removing the subfolders from the sidebar.

**Reply by alexanderloewe** [reply]
@brandonkelly: I had read the post, yet I find your old way of presenting files, faster, clearer and easier, especially for people who have a rather greater understanding of data management themselves.

Why not give the user the choice which view he would like to use?
